server/src/printers/TestMakefilesPrinter.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Huawei Technologies Co., Ltd. 2012-2021. All rights reserved.
+ */
+
+#include "TestMakefilesPrinter.h"
+#include "utils/FileSystemUtils.h"
+
+#include <utility>
+#include <utils/MakefileUtils.h>
+
+namespace printer {
+    fs::path get_shared(fs::path path) {
+        return Paths::addSuffix(Paths::addPrefix(path, "__"), "_shared");
+    }
+
+    fs::path get_obj(fs::path path) {
+        return Paths::addSuffix(Paths::addPrefix(path, "__"), "_obj");
+    }
```

Comment:
Use _lowerCamelCase_ for names of functions, please. In this case, `getObjectFileForLinkage` or smth like that is more clear than `get_obj`.

Reply:
rename getMakefilePathForObject